### DISCUSSION TITLE
MenuItem broken?

### DISCUSSION THREAD

**Discussion by drei97** [AvaloniaUI/Avalonia#14637] [Q&A]
Hi,

I recently started working on a project again which was 11.0.5 and had it updated to 11.0.9.

Now the main menu is broken. No sub item of 2nd order of a MenuItem is coming up.

```
        <Menu Grid.Row="0">
            <MenuItem Header="File">
                <MenuItem Header="Settings">
                    <MenuItem Header="Paths" Command="{Binding CommandClick}"
                              CommandParameter="Settings" />
                </MenuItem>
                <Separator />
...
```
Going on "File" opens "Settings", but "Paths" is not accessible.

Must be a bug?

Any idea when it was introduced, that a can downgrade? Without menu the program is not really useful.

**Comment by sn4k3** [comment]
It's a bug introduced on 10.0.8 and continues with 10.0.9: You need to revert for 10.0.7

https://github.com/AvaloniaUI/Avalonia/issues/14568

**Reply by drei97** [reply]
will it be fixed in 11.0.10?

For now I can deal with a workaround.
